neural_compressor/training.py

```diff
@@ -17,9 +17,9 @@
 
 import copy
 from .conf.pythonic_config import Config
-from .config import DistillationConfig, PruningConfig, QuantizationAwareTrainingConfig
```

Comment:
Why WeightPruningConfig is in pruning file but config file?

Reply:
circular import issue will occur. Also, this makes the pruning module self-contained.